template/build.sh

```diff
@@ -18,7 +18,7 @@ if test ! -d $buildFolder; then
     
     localZipFile="$tempFolder/korebuild.zip"
     
-    wget -O $localZipFile $koreBuildZip 2>/dev/null || curl -o $localZipFile --location $koreBuildZip /dev/null
+    wget -O $localZipFile $koreBuildZip 2>/dev/null || curl -o $localZipFile --location $koreBuildZip 2>/dev/null
```

Comment:
It does not break really. Even though it shows an error message it will download what we want. Still it would be good to fix this - at the moment `/dev/null` is being treated as an url and you may see this: `curl: (3) <url> malformed`. (Granted almost no one see even this because everyone seems to have wget on their Macs)